packages/lesmis/src/routes/AdminPanelRoutes.js

```diff
@@ -34,36 +34,32 @@ import {
   DraftSurveyResponsesView,
 } from '../views/AdminPanel/SurveyResponsesView';
 
-const ADMIN_URL = '/admin';
```

Comment:
I'm also taking this chance to fix a regression in the admin panel url that was introduced during internationalisation. The admin segment is now added upstream